code/modules/clothing/head/f13head.dm

```diff
@@ -222,13 +222,20 @@
 	dynamic_hair_suffix = ""
 	dynamic_fhair_suffix = ""
 	speechspan = SPAN_ROBOT //makes you sound like a robot
+	heat_protection = HEAD
+	max_heat_protection_temperature = FIRE_HELM_MAX_TEMP_PROTECT
+	cold_protection = HEAD
+	min_cold_protection_temperature = FIRE_HELM_MIN_TEMP_PROTECT
 //	darkness_view = 128
 //	lighting_alpha = LIGHTING_PLANE_ALPHA_NV_TRAIT
 	var/on = FALSE
 	var/brightness_on = 5
 	var/power_on = 1
 	var/emped = 0
 	var/requires_training = TRUE
+	var/armor_block_chance = 0
+	var/list/protected_zones = list(BODY_ZONE_HEAD)
+	var/deflection_chance = 0
```

Comment:
I would probably rework this to be based on armour penetration; if something hits with high enough force and low enough armour penetration, that force should be redirected into a ricochet.

~~Alternatively, remove it entirely. Fallout 2 doesn't have deflection.~~